packages/core/src/components/card/card.tsx

```diff
@@ -104,40 +84,38 @@ export class Card implements ComponentInterface {
 
   render() {
     const isSelectable = this.selectableSlottedElement !== null;
-    const hasImage = this.imageSlottedElement !== null;
 
     return (
-      <Host onClick={(e: MouseEvent) => this.clickEventHandler(e)} is-selectable={isSelectable} has-image={hasImage}>
+      <Host is-selectable={isSelectable}>
         <div class="dso-card-selectable" hidden={!isSelectable}>
           <slot name="selectable" />
         </div>
-        <div class="dso-card-image" hidden={!hasImage}>
-          <slot name="image" />
-        </div>
         <div class="dso-card-heading">
-          {this.headingSlottedElement instanceof HTMLAnchorElement || !this.href ? (
-            <slot name="heading" />
-          ) : (
-            ((!this.mode || !["download", "extern"].includes(this.mode)) && (
-              <a href={this.href} class="heading-anchor">
-                <slot name="heading" />
-                <dso-icon icon="chevron-right"></dso-icon>
-              </a>
-            )) ||
+          {((!this.mode || !["download", "extern"].includes(this.mode)) && (
```

Comment:
Zullen we deze voorwaarde in een beschrijvende getter of method plaatsen?

Reply:
Is dit niet eigenlijk een `else` achtig ding? En ligt het dan niet meer voor de hand om die aan het einde te plaatsen?